ui/e2e/src/tests/panelEditor.spec.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { test, expect } from '../fixtures/dashboardTest';
+import { PanelEditor } from '../pages/PanelEditor';
+
+test.use({
+  dashboardName: 'Panels',
+});
+
+test.describe('Dashboard: Panel Editor', () => {
+  test('should show discard changes confirmation dialog', async ({ dashboardPage }) => {
```

Comment:
Could you clarify the use case a little bit. Maybe something like "requires confirmation to discard changes", so the full test reads "Panel editor requires confirmation to discard changes."

Related: could you also add a test case for the reason you're making this PR. Maybe something like "can be closed without confirmation if nothing has changed"?